oio/cli/admin/item_repair.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (C) 2019 OpenIO SAS, as part of OpenIO SDS
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+from cliff import lister
+from logging import getLogger
+
+
+from oio.blob.rebuilder import BlobRebuilder
+from oio.cli.admin.common import ObjectCommandMixin
+
+
+class ObjectRepair(ObjectCommandMixin, lister.Lister):
+    """
+    Repair all missing, lost or corrupt chunks of object.
+    """
+
+    log = getLogger(__name__ + '.ObjectRepair')
+    columns = ('Chunk', 'Status')
+    rebuilder = None
+
+    def get_parser(self, prog_name):
+        parser = super(ObjectRepair, self).get_parser(prog_name)
+        self.patch_parser(parser)
+        return parser
+
+    def _take_action(self, parsed_args):
+        conf = dict()
+        conf['namespace'] = self.app.options.ns
```

Comment:
I suggest using `self.app.client_manager.client_conf`. This allows running (some) tools without loading the namespace configuration file, provided enough parameter have been specified on the command line.